#human.msg

float32 top_left_x
float32 top_left_y
float32 bottom_right_x
float32 bottom_right_y
